database/migrations/2021_03_19_092900_add_unique_index_to_language_codes.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUniqueIndexToLanguageCodes extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::table('languages', function (Blueprint $table) {
+            $table->unique(['code']);
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down()
+    {
+        Schema::table('languages', function (Blueprint $table) {
+            $table->dropIndex(['code']);
```

Comment:
Pretty sure when dropping indexes you have to use the full index name.

Reply:
Not when you pass an array I believe